### DISCUSSION TITLE
When I try to add a Public Github Repo on ArgoCD, I get an error.

### DISCUSSION THREAD

**Discussion by 0n1dev** [argoproj/argo-cd#9414] [Q&A]
When I try to add a Public Github Repo on ArgoCD, I get an error.
level=error msg="finished unary call with code unknown" error="rpc error: code = unknown
I think a timeout is occurring, so I would like to know how to fix it.

**Comment by rishabh625** [comment]
Hi @0n1dev : Did u make sure that repo-server is able to reach ti github repo , isn't there any network issue
